package.json

```diff
@@ -61,19 +61,21 @@
     "ajv-formats": "^2.1.1",
     "chai": "^4.3.4",
     "mocha": "^9.1.1",
+    "npm-run-all": "^4.1.5",
     "prettier": "^2.4.1",
     "source-map-support": "^0.5.20",
     "tslint": "^6.1.3"
   },
   "scripts": {
-    "prepublishOnly": "yarn build",
-    "test": "yarn build && mocha -t 5000 --require source-map-support/register dist/test",
+    "prepare": "run-s build",
+    "test": "run-s build test:mocha",
+    "test:mocha": "mocha -t 5000 --require source-map-support/register dist/test",
     "debug": "node --inspect=19248 --inspect-brk -r ts-node/register typescript-json-schema-cli.ts",
     "docs": "./update-docs.js",
     "run": "ts-node typescript-json-schema-cli.ts",
     "build": "tsc -p .",
     "lint": "tslint --project tsconfig.json -c tslint.json --exclude '**/*.d.ts'",
-    "style": "yarn prettier --write *.js *.ts test/*.ts",
+    "style": "prettier --write *.js *.ts test/*.ts",
```

Comment:
Doesn't this mean prettier has to be global?

Reply:
No, it is installed as a dev dependency, it will just work.